### PR TITLE
V5 Docs - `permutePath`

### PR DESCRIPTION
**Pull request by colincasey**
Rendered pages:
- https://github.com/salesforce/tough-cookie/blob/v5-docs-permutepath/api/docs/tough-cookie.permutepath.md

---
Main PR:
- #384

### CODE REVIEW

File: api/docs/tough-cookie.permutepath.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+</td><td>
+
+the path to generate permutations for
+
+
+</td></tr>
+</tbody></table>
+**Returns:**
+
+string\[\]
```
Comment (wjhsf) [inline api/docs/tough-cookie.permutepath.md:52]: This doesn't seem to be formatted correctly. 🫤

File: api/docs/tough-cookie.permutepath.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+## Example
+
+
+```
```
Comment (wjhsf) [inline api/docs/tough-cookie.permutepath.md:57]: Examples seem to be missing the `ts` language tag for syntax highlighting. Is there any way to add it?

**Review by wjhsf — APPROVED** [review]

File: api/docs/tough-cookie.permutepath.md (hunk 1 above)
Comment (colincasey) [reply api/docs/tough-cookie.permutepath.md:52]: I noticed this too and will file a bug with `@microsoft/api-documenter` if one doesn't already exist.

File: api/docs/tough-cookie.permutepath.md (hunk 2 above)
Comment (colincasey) [reply api/docs/tough-cookie.permutepath.md:57]: Yes, it looks like adding `ts` to the block is respected in the output. 

I believe I missed this for several of the PRs. Once these are all merged into the `v5-docs` branch I'll need to do a final bit of cleanup (e.g.; removing the doc `.gitignore`, verifying there are no warnings about missing types from our public api, etc.) so I'll add the following to the list:
- annotate examples with `ts` for syntax highlighting

**Pull request merged**